packages/xstate-react/test/useMachine.test.tsx

```diff
@@ -647,4 +647,57 @@ describe('useMachine (strict mode)', () => {
     );
     done();
   });
+
+  // https://github.com/davidkpiano/xstate/issues/1334
+  it('should work with a rehydrated state', (done) => {
+    const testMachine = Machine({
+      id: 'app',
+      initial: 'idle',
+      states: {
+        idle: {
+          on: {
+            START: 'doingStuff'
+          }
+        },
+        doingStuff: {
+          id: 'doingStuff',
+          after: {
+            100: 'idle'
+          }
+        }
+      }
+    });
+
+    const persistedState = JSON.stringify(testMachine.initialState);
+
+    let currentState = testMachine.resolveState(
```

Comment:
is this value used anywhere? from what i see it gets overridden pretty immediately (when rendering `Test`)